sdk/communication/azure-mgmt-communication/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.0.0b2 (2020-09-22)
+
+* Updates for track 2
```

Comment:
The release note makes no sense to SDK users. Please refer to https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/resources/azure-mgmt-resource/CHANGELOG.md#1500b1-2020-06-17 which transit from track1 to track2.

Reply:
Changelog has been updated with additional information